lib/aur-search

```diff
@@ -106,7 +106,7 @@ trap_exit() {
 }
 
 usage() {
-    printf >&2 'usage: %s: [-adimnqrsv] [-k key] pkgname...\n' "$argv0"
+    plain 'usage: %s [-adimnqrsv] [-k key] pkgname...' "$argv0" >&2
```

Comment:
Maybe add a comment in the relevant `[[ -t 1 ]]` section too (re. #473, as a reminder to fix it in a different PR)

Reply:
Out of scope here. I'll open an issue to keep track of it.